modules/otel-config-instrumentation.adoc

```diff
@@ -0,0 +1,331 @@
+// Module included in the following assemblies:
+// 
+// * otel/otel-instrumentation.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="otel-instrumentation-config_{context}"]
+= OpenTelemetry instrumentation configuration options
+
+The {OTELName} can inject and configure the OpenTelemetry auto-instrumentation libraries into your workloads. Currently, the project supports injection of the instrumentation libraries from Go, Java, Node.js, Python, .NET, and the Apache HTTP Server (`httpd`).
+
+Auto-instrumentation in OpenTelemetry refers to the capability where the framework automatically instruments an application without manual code changes. This enables developers and administrators to get observability into their applications with minimal effort and changes to the existing codebase.
+
+[IMPORTANT]
+====
+The {OTELName} Operator only supports the injection mechanism of the instrumentation libraries but does not support instrumentation libraries or upstream images. Customers can build their own instrumentation images or use community images.
+====
+
+== Instrumentation options
```

Comment:
Is there a sentence that could introduce this section?

Reply:
Added. Thank you.